service/src/main/java/io/camunda/service/UserServices.java

```diff
@@ -54,6 +56,13 @@ public UserServices withAuthentication(final Authentication authentication) {
         brokerClient, securityContextProvider, userSearchClient, authentication, passwordEncoder);
   }
 
+  public Optional<UserEntity> getUserByUsername(final String username) {
```

Comment:
❓ I cannot find where is the method is used?

Reply:
It's gone now. I had used that for (incorrectly) mapping oidc tokens, but I ended up deleting that code.